### ISSUE TITLE
Generate test cubes automatically

### ISSUE THREAD

**Issue by cdeil** [radio-astro-tools/spectral-cube#109]
I'm having a look at `spectra-cube` and noticed that `python setup.py test` fails because the test cube data is missing.

From the `.travis.yml` file I see that I should be doing this:

```
cd spectral_cube/tests/data ; make ; cd ../../../
```

Would it make sense to generate the test data automatically?

**Comment by keflavich** [comment]
How safe is it to just add a few lines like:
```
os.chdir('spectral_Cube/tests/data')
os.system('make')
```
in `setup.py`?

**Comment by keflavich** [comment]
Solved by #427 - we just do it now.
